### DISCUSSION TITLE
Add a Notes Field

### DISCUSSION THREAD

**Discussion by RamenJunkie** [FreshRSS/FreshRSS#6058] [Ideas]
It would be really cool if you could add user notes in a little field, similar to how tags are added, on articles.  They could help for finding things in search, or for remembering why you saved/tagged an article.  

Also, somewhat for my own use case, I have some scripts that make digests of articles I've saved based on tags each day, and having these notes in the RSS feed so I could add this commentary would be really useful.  But I think in general it could be a useful feature.

**Comment by Alkarex** [comment]
Yes, agree. This is also quite high on my own TODO list, especially after the new RSS / HTML sharing https://github.com/FreshRSS/FreshRSS/pull/6052 has landed.
It would be good to search in our existing issues for similar suggestions / requests.
